doc/topics/releases/boron.rst

```diff
@@ -4,23 +4,42 @@
 Salt Release Notes - Codename Boron
 ===================================
 
+Backwards-incompatible Changes
+==============================
+
+- The default path for the :conf_master:`extension_modules` master config
+  option has been changed.  Prior to this release, the location was a directory
+  named ``extmods`` in the Salt cachedir. On most platforms, this would put the
+  :conf_master:`extension_modules` directory in ``/var/cache/salt/extmods``.
+  It has been moved one directory down, into the master cachedir. On most
+  platforms, this is ``/var/cache/salt/master/extmods``.
```

Comment:
Wondering if we should add a "most people don't have to worry" note. "For most people, this will have no affect on your setup. However, if you were manually placing files into this directory to be loaded by the master, you will have to change the directory you use." or something.